xml/operations-objectstorage-swift_cli.xml

```diff
@@ -20,8 +22,9 @@
   Then you can use your preferred editor to edit swiftuser.osrc so you can
   authenticate using the <literal>OS_USERNAME</literal>,
   <literal>OS_PASSWORD</literal>, and <literal>OS_PROJECT_NAME</literal> you
-  wish to use. For example, if you would like to use the <literal>demo</literal>
-  user that is created automatically for you, then it might look like this:
+  wish to use. For example, if you would like to use the
+  <literal>demo</literal> user that is created automatically for you, then it
+  might look like this:
```

Comment:
remove "then it might look like this" - it works with just "for example" :) Or maybe switch so it reads...
"The following example displays the configuration if you want to use the demo user that is created automatically for you:"